python/oneflow/nn/graph/graph.py

```diff
@@ -447,10 +450,12 @@ def _compile(self, *args):
             assert not self._is_compiled, (
                 "nn.Graph " + self._name + " has already been compiled."
             )
-
+            t0 = time.clock()
             eager_outputs = self._build_graph(*args)
-
-            self._print(0, 0, self._shallow_repr() + " end building graph.")
+            t1 = time.clock()
+            self._print(0, 0, self._shallow_repr()  + "time consumed to complete build graph:"
```

Comment:
```suggestion
            self._print(0, 0, self._shallow_repr()  + "time for building graph:"
```
看最终输出信息，因为之前已经说过了是“building graph”，所以后面是否尽量简洁就可以。

Reply:
已经按照意见修改